wallet/src/de/schildbach/wallet/service/BlockchainServiceExt.kt

```diff
@@ -0,0 +1,13 @@
+package de.schildbach.wallet.service
+
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.asCoroutineDispatcher
+import kotlinx.coroutines.launch
+import java.util.concurrent.Executors
+
+fun BlockchainServiceImpl.initCoinJoin() {
+    val scope = CoroutineScope(Executors.newSingleThreadExecutor().asCoroutineDispatcher())
+    scope.launch {
+        coinJoinService.configureMixing(true)
+    }
+}
```

Comment:
I would like to get rid of this extension.  `CoinJoinService` is largely self-contained and should "configure itself" rather than be told to.